### DISCUSSION TITLE
Gauge gradle plugin release

### DISCUSSION THREAD

**Discussion by janox1x2x3** [getgauge/gauge#1910] [General]
Hey, 

can you please release new version of gauge gradle plugin... It seems like [this](https://github.com/getgauge/gauge-gradle-plugin/pull/47) got merged right after the last release, but the version with that fix never got released.

Thank you

**Comment by sriv** [comment]
Done, you should now be able to use version `1.8.2` of the gauge gradle plugin.

**Reply by janox1x2x3** [reply]
thanks man
